src/Explorer/Quickstart/QuickstartGuide.tsx

```diff
@@ -0,0 +1,294 @@
+import {
+  DefaultButton,
+  Icon,
+  IconButton,
+  Image,
+  IPivotItemProps,
+  Pivot,
+  PivotItem,
+  PrimaryButton,
+  Stack,
+  Text,
+  TextField,
+} from "@fluentui/react";
+import React, { useState } from "react";
+import Youtube from "react-youtube";
+import Pivot1SelectedIcon from "../../../images/Pivot1_selected.svg";
+import Pivot2Icon from "../../../images/Pivot2.svg";
+import Pivot2SelectedIcon from "../../../images/Pivot2_selected.svg";
+import Pivot3Icon from "../../../images/Pivot3.svg";
+import Pivot3SelectedIcon from "../../../images/Pivot3_selected.svg";
+import Pivot4Icon from "../../../images/Pivot4.svg";
+import Pivot4SelectedIcon from "../../../images/Pivot4_selected.svg";
+import Pivot5Icon from "../../../images/Pivot5.svg";
+import Pivot5SelectedIcon from "../../../images/Pivot5_selected.svg";
+import CompleteIcon from "../../../images/QuickstartComplete.svg";
+import { ReactTabKind, useTabs } from "../../hooks/useTabs";
+
+enum GuideSteps {
+  Login,
+  NewTable,
+  DistributeTable,
+  LoadData,
+  Query,
+}
+
+export const QuickstartGuide: React.FC = (): JSX.Element => {
+  const [currentStep, setCurrentStep] = useState<number>(0);
+  const newTableCommand = `CREATE TABLE github_users
+(
+user_id bigint,
+url text,
+login text,
+  .....`;
+  const distributeTableCommand = `SELECT create_distributed_table('github_users', 'user_id');
+SELECT create_distributed_table('github_events', 'user_id');`;
+  const loadDataCommand = `-- download users and store in table
+
+COPY github_users FROM PROGRAM 'curl https://examples.citusdata.com/
+users.csv' WITH (FORMAT CSV)`;
+  const queryCommand = `-- Find all events for a single user.
+-- (A common transactional/operational query)
+
+SELECT created_at, event_type, repo->>'name' AS repo_name
+FROM github_events
+WHERE user_id = 3861633;`;
+
+  const onCopyBtnClicked = (selector: string): void => {
+    const textfield: HTMLInputElement = document.querySelector(selector);
+    textfield.select();
+    document.execCommand("copy");
+  };
+
+  const getPivotHeaderIcon = (step: number): string => {
+    if (step === 0) {
+      return Pivot1SelectedIcon;
```

Comment:
Is this redundant considering the switch statement below ?

Reply:
Yes thanks for catching this. I'll remove it.